### PR TITLE
📦 introduce new bundle company-users-bulk-rest-api-business-central-connector

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: bundles/company-users-bulk-rest-api-business-central-connector/src/FondOfImpala/Zed/CompanyUsersBulkRestApiBusinessCentralConnector/Persistence/CompanyUsersBulkRestApiBusinessCentralConnectorRepository.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        return $collection->setCompanies(new ArrayObject($companyTransfers));
+    }
+
+    /**
+     * @param array $debtorNumbers
+     *
+     * @return array<int, \Generated\Shared\Transfer\CompanyUsersBulkCompanyTransfer>
+     */
+    protected function getCompanyTransfers(array $debtorNumbers): array
```
Comment (daniel-rose) [inline bundles/company-users-bulk-rest-api-business-central-connector/src/FondOfImpala/Zed/CompanyUsersBulkRestApiBusinessCentralConnector/Persistence/CompanyUsersBulkRestApiBusinessCentralConnectorRepository.php:38]: Maybe `getCompanyMapByDebtorNumbers` is a better name.

File: bundles/company-users-bulk-rest-api-business-central-connector/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+phpcs:
+	./vendor/bin/phpcs --standard=./vendor/spryker/code-sniffer/Spryker/ruleset.xml ./src/FondOfImpala/* ./tests/*
+
+.PHONY: phpcbf
+phpcbf:
+	./vendor/bin/phpcbf --standard=./vendor/spryker/code-sniffer/Spryker/ruleset.xml ./src/FondOfImpala/* ./tests/*
+
+.PHONY: phpstan
+phpstan:
+	./vendor/bin/phpstan --memory-limit=-1 analyse ./src/FondOfImpala ./tests
```
Comment (daniel-rose) [inline bundles/company-users-bulk-rest-api-business-central-connector/Makefile:15]:
```suggestion
	./vendor/bin/phpstan --memory-limit=-1 analyse ./src/FondOfImpala
```

File: bundles/company-users-bulk-rest-api-business-central-connector/Makefile
Comment (daniel-rose) [inline bundles/company-users-bulk-rest-api-business-central-connector/Makefile:1]: You need a `clean`-target. Look at https://github.com/fond-of-impala/fond-of-impala/blob/bec6c9ca205877e47a9a2569cb35c6be472e1336/bundles/conditional-availability/Makefile#L24C9-L24C14.

File: bundles/company-users-bulk-rest-api-business-central-connector/composer.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "email": "gengjozsef86@gmail.com"
+    },
+    {
+      "name": "Julian Hyatt",
+      "email": "dev@jzit.de"
+    }
+  ],
+  "require": {
+    "php": ">=8.0",
+    "fond-of-impala/company-users-bulk-rest-api-extension": "*",
```
Comment (daniel-rose) [inline bundles/company-users-bulk-rest-api-business-central-connector/composer.json:17]: Use version please... (like ^1.0.0)

**Review by daniel-rose — APPROVED** [review]

File: bundles/company-users-bulk-rest-api-business-central-connector/composer.json (hunk 3 above)
Comment (julianzimmermann) [reply bundles/company-users-bulk-rest-api-business-central-connector/composer.json:17]: the extension has no release yet. when all packages are in master i can make the releases and add correct version

**Pull request merged**